docs/digger/installation/digger-install-intro.asciidoc

```diff
@@ -1,15 +1,16 @@
 == Digger Installation Introduction
 //Please DO NOT use 'Overview' as a section heading anywhere http://stylepedia.net/#sect-Red_Hat_Technical_Publications-Writing_Style_Guide-Overall_Book_Design-Unused_Heading_Titles
 
-The AeroGear Digger project allows you to setup a service builds mobile applications using OpenShift containers.
+The AeroGear Digger project allows you to setup a service that has the capability to build mobile applications using OpenShift containers.
```

Comment:
I resolved this in my pr also. Happy with either wording https://github.com/aerogear/aerogear.org/pull/666/files#diff-9b0673431a0497bea166bec4b33245aeR3

Reply:
:+1. I imagine yours will be merged first so I will resolve it when I rebase